zerver/lib/send_email.py

```diff
@@ -12,10 +12,10 @@
 import datetime
 from email.utils import parseaddr, formataddr
 import logging
-import ujson
+import jsonpickle
```

Comment:
As a policy matter, we avoid using pickles.  We should figure out a better way to encode this.